### ISSUE TITLE
Wrong strand info in the ccds.15.grch37p13.extended.bed?

### ISSUE THREAD

**Issue by biocyberman** [robinandeer/chanjo#180]
I don't know why the `strand` values for all exons are `-` minus.
```
coverage ➤gawk '{print $6}' ccds.15.grch37p13.extended.bed |sort|uniq 
-             
```

**Comment by robinandeer** [comment]
hmm weird, I don't use this information in chanjo so I guess I was thinking I'd put some dummy data to follow the specifications

Might be so stupid that I meant "-" to mean "nothing" without really thinking haha

**Comment by biocyberman** [comment]
Yeah, so it should be '.' (dot) instead. I use the bed file in sambamba and other work. So it is better to have it right. 

Anyway, to avoid troubles with dependencies of the python solution, I wrote a bash+gawk script here:
https://github.com/biocyberman/bscripts/blob/master/forBED/CCDStoBED.sh

**Comment by robinandeer** [comment]
Oh that's awesome! I will make sure to link to it :D
